consumer/dragonboard/qualcomm-robotics-rb5/installation/README.md

```diff
@@ -0,0 +1 @@
+---
title: Installation for Qualcomm Robotics RB5 development kit
permalink: /documentation/consumer/dragonboard/qualcomm-robotics-rb5/installation/
---
# Installation

This guide will help you choose and install an operating system and installation method on your Qualcomm Robotics RB5 development kit.

To appropriately follow this installation guide you will need to:

- Download necessary files
- Choose host machine
- Follow flashing instructions

***

### Fastboot Method

Fastboot is supported by the board and can be used for installs.

This method requires the following hardware:

- Qualcomm Robotics RB5 development kit with power supply
- Host machine (Linux, Mac OS X, or Windows)
- USB to USB Type C cable
- USB Mouse and/or keyboard (not required to perform flash)
- HDMI Monitor with full size HDMI cable (not required to perform flash)

Go to the [Downloads page](../downloads/) to get your bootloader, boot image, and root file system image (rootfs).

Choose host machine

- [Linux](linux-fastboot.md)

***

### Board Recovery

Attempt to recover a board from a bad flash resulting in a soft-bricked board.
- [Board Recovery](board-recovery.md)

***

### Third Party OS

- [LE OS Installation](le.md)


```

Comment:
formatting needs to be fixed, weird `^M` characters appearing in-place of  regular `<enter>/<return>`